test/integration/targets/apt/tasks/apt.yml

```diff
@@ -140,6 +140,40 @@
 - name: uninstall hello with apt
   apt: pkg=hello state=absent purge=yes
 
+# verify that apt is handling multi-arch systems properly
```

Comment:
In order to improve readability, test tasks related to multi-arch could be moved in a dedicated YAML file and used here using an `include` task.